src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div id="Calcu">
+      <div className="rowA Display">011</div>
+      <div className="rowB">
+        <button type="button" className="Calc-button">AC</button>
+        <button type="button" className="Calc-button">+/-</button>
+        <button type="button" className="Calc-button">%</button>
+        <button type="button" className="Calc-button Calcu-operator">&#247;</button>
+      </div>
+      <div className="rowC">
+        <button type="button" className="Calc-button">7</button>
+        <button type="button" className="Calc-button">8</button>
+        <button type="button" className="Calc-button">9</button>
+        <button type="button" className="Calc-button Calcu-operator">&#215;</button>
+      </div>
+      <div className="rowD">
+        <button type="button" className="Calc-button">4</button>
+        <button type="button" className="Calc-button">5</button>
+        <button type="button" className="Calc-button">6</button>
+        <button type="button" className="Calc-button Calcu-operator">-</button>
+      </div>
+      <div className="rowE">
+        <button type="button" className="Calc-button">1</button>
+        <button type="button" className="Calc-button">2</button>
+        <button type="button" className="Calc-button">3</button>
+        <button type="button" className="Calc-button Calcu-operator">+</button>
+      </div>
+      <div className="rowF">
+        <button type="button" className="Calc-button-zero">0</button>
+        <button type="button" className="Calc-button">.</button>
+        <button type="button" className="Calc-button Calcu-operator">=</button>
+      </div>
+    </div>
```

Comment:
👍 Great job with your calculator! However, in React, it's a common practice to split your code into several components to make it more organized, maintainable, and reusable. This approach aligns with the project requirements, which suggest having at least one child component for the `Calculator` component.

Here are a couple of ways you can achieve this:

- [ ] One way is to create a new button component since you are repeating the same button while only changing the value. Another way is to have a separate component for all the buttons, which will be easy to handle, and a separate component for the display. There are many possibilities to split the code, but it's important to remember that doing so will help you maintain and improve your reusable components. I think you should follow the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component) and have at least one child component.

Here's an example of how you could structure your code:

```jsx
// Calculator.js
import React from 'react';
import Button from './Button'; // Import the Button component
import Display from './Display'; // Import the Display component

const Calculator = () => {
  // Calculator component logic here
  return (
    <div className="calculator">
      <Display />
      <div className="buttons">
        <Button value="1" />
        <Button value="2" />
        {/* Add more buttons */}
      </div>
    </div>
  );
};

export default Calculator;
```

By following this approach, you'll enhance the maintainability and reusability of your components while also adhering to the project requirements. Keep up the excellent work! 👍